data/projects/documentation_extraction.md

```diff
@@ -0,0 +1,33 @@
+Code needs documentation. Most programming languages have some form of inline 
+documentation standard.
+
+For python, its [docstrings](https://www.python.org/dev/peps/pep-0257/)
+
+The aim of the project is to develop a language independent analyzing routine 
+that will find all documentation strings in a file, parse them into specific 
+groups and check them against a specified documentation style given by the 
+user. The functionality can be extended so that the analyzing routine provides
+a patch that re-formats the documentation correctly, check for grammatical 
+errors and perform other aesthetical fixes.
+
+
+#### Milestones
+
+##### GSOC 2017 COMMUNITY BONDING
+
+* Learn about how different languages have their own documentation standards.
+* Learn about the work already done.
+* Design an effecient parsing system.
```

Comment:
Does this need a cEP? or just a concept?
The first two milestones are not realy measurable. So maybe just have "A concept for the design of an efficient parsing system exists", which implies the first two points